kraken/kubernetes/client.py

```diff
@@ -23,6 +23,32 @@ def initialize_clients(kubeconfig_path):
         sys.exit(1)
 
 
+def get_host() -> str:
+    """Returns the Kubernetes server URL"""
+    return client.configuration.Configuration.get_default_copy().host
+
+
+def get_clusterversion_string() -> str:
+    """Returns clusterversion status text on OpenShift, empty string on other distributions"""
```

Comment:
Small nit, some of the functions if they have a description/comment about what they do are above the function definition. Looks good to me other than that

Reply:
I tried to follow the [docstring conventions](https://peps.python.org/pep-0257/) and think we should continue with that as we refactor some of the code since it will help us with documentation long-term.

Reply:
Makes sense! Thanks